app/views/ThreadMessagesView/filters.ts

```diff
@@ -0,0 +1,5 @@
+export enum FILTER {
```

Comment:
Now we're migrating to Typescript, we don't need to use uppercase to identify constants.
Can you rename this to Filter and same for the values?
It would be used like `Filter.All`, `Filter.Following`, `Filter.Unread`